src/formatDate.js

```diff
@@ -8,7 +8,25 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateComponents = date.split(fromFormat[3]);
+  const result = [];
+
+  result[toFormat.indexOf('DD')] = dateComponents[fromFormat.indexOf('DD')];
+  result[toFormat.indexOf('MM')] = dateComponents[fromFormat.indexOf('MM')];
+
+  let year = fromFormat.includes('YYYY')
```

Comment:
Using `includes` to check for the presence of 'YYYY' or 'YY' in the format array is a good approach. This ensures that the function can adapt to both year formats.